contrib/guix/manifest.scm

```diff
@@ -168,7 +168,7 @@ desirable for building Bitcoin Core release binaries."
          (pthreads-xlibc mingw-w64-x86_64-winpthreads)
          (pthreads-xgcc (make-gcc-with-pthreads
                          (cross-gcc target
-                                    #:xgcc (make-ssp-fixed-gcc base-gcc)
+                                    #:xgcc (package-with-extra-configure-variable (make-ssp-fixed-gcc base-gcc) "--with-newlib" "no")
```

Comment:
Could you split this out something like:
```scheme
;; why we are doing this
(define (make-gcc-without-newlib gcc)
  (package-with-extra-configure-variable gcc "--with-newlib" "no"))
```

```suggestion
                                    #:xgcc (make-gcc-without-newlib (make-ssp-fixed-gcc base-gcc))
```

Reply:
Thanks! [Updated](https://github.com/bitcoin/bitcoin/pull/24348#issuecomment-1040429493).